docs/Troubleshooting.md

```diff
@@ -3,6 +3,8 @@
 Make sure:
 
 - You have at least Node version 6 installed
+- You have Yarn installed
+- You're visiting: [http://localhost:8000]()
```

Comment:
This is a broken link. There's no reason to put http://localhost:8000 in brackets, but once you do it like this, it breaks the link. So remove the brackets.